build-tools.Dockerfile

```diff
@@ -3,8 +3,17 @@ ARG DEBIAN_VERSION=bookworm
 FROM debian:bookworm-slim AS pgcopydb_builder
 ARG DEBIAN_VERSION
 
+# Use strict mode for bash to catch errors early
+SHELL ["/bin/bash", "-euo", "pipefail", "-c"]
+
+# Some complexity with `echo`:
+# In Debian images for RUN used `/bin/sh`
+# and it treats `\n` as newline by default.
+# If you try it with `bash`, then you need `-e` flag.
+# For other images, like `alpine/curl` for example,
+# They use `bash` by default, so there we will use `-e` for echo.
```

Comment:
This comment is a bit confusing now. The point is that we explicitly use bash, and because we use bash, we can and must use `echo -e`.

Reply:
fixed, thanks